Project/Assets/Prairie/Framework/Script/Interaction/Annotation.cs

```diff
@@ -221,4 +223,33 @@ void Update()
             }
         }
     }
+
+	void OnTriggerEnter(Collider other)
+	{
+
+		// ensure we're being triggered by a player
+		player = other.gameObject.GetComponent<FirstPersonInteractor> ();
+		if (player == null)
```

Comment:
This is a good method for tracking the player toolbar, although I think we need some additional checking.

I don't remember what we decided as a group, but as is, any `Annotation` sharing a game object with a `Collider` (set `isTrigger`) will have area annotation behavior.